docs/DeveloperGuide.md

```diff
@@ -721,3 +721,16 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+## **Appendix: Planned Enhancements**
+**Team size**: 5
+
+1. **Make find command for searching module-role more general, such that user can find all contacts with roles related to a specified module.
```

Comment:
I think the phrasing can be more accurate, since after merging PR #320, we will be able to find different roles of the same module, using:
`find r/cs1101s r/cs1101s-ta`

```suggestion
1. **Allow the user to find people with any role associated with a module, by leaving out the role in the command.**
```

Reply:
To address #273, can add partial match to find module-role